src/waitFor.ts

```diff
@@ -203,24 +200,10 @@ export default async function waitFor<T>(
 
   let result: T;
 
-  //$FlowFixMe: `act` has incorrect flow typing
   await act(async () => {
     result = await waitForInternal(expectation, optionsWithStackTrace);
   });
 
-  //$FlowFixMe: either we have result or `waitFor` threw error
+  // @ts-ignore either we have result or `waitFor` threw error
   return result;
```

Comment:
I think you can use `!` operator to force TS to trust you it's always valid. You can keep the comment though.